linux/completions/mw.bash

```
@@ -9,16 +9,29 @@ _mw_complete() {
       ;;
+    memory)
+      [ "$COMP_CWORD" -eq 2 ] && COMPREPLY=( $(compgen -W "stale supersede" -- "$cur") )
+      ;;
+    hooks)
+      [ "$COMP_CWORD" -eq 2 ] && COMPREPLY=( $(compgen -W "install uninstall remove" -- "$cur") )
+      ;;
+    integrate)
+      [ "$COMP_CWORD" -eq 2 ] && COMPREPLY=( $(compgen -W "hermes" -- "$cur") )
+      ;;
+    sync-mempalace)
+      COMPREPLY=( $(compgen -W "--wing --limit --dry-run" -- "$cur") )
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)